RELEASE_NOTES.md

```diff
@@ -2,6 +2,7 @@
 
 ## New in git-machete 3.11.0
 
+- added: `git machete help config` help topic and sphinx documentation page for config keys and environment variables
 - added: boolean git config key `machete.worktree.useTopLevelMacheteFile` that enables per-worktree machete definition file (as opposed to a single central `.git/machete` file for all worktrees)
```

Comment:
Actually... this is pretty misleading (yeah, that was my suggestion :trollface:). I mean, it's technically true... but the problem with this formulation is:
* we mention a key whose name says _use **top-level** machete file_
* and then we say that this key _enables **per-worktree** machete file_

To make it more clear, I'd change it to:
```suggestion
- added: boolean git config key `machete.worktree.useTopLevelMacheteFile` for switching the machete file location for worktrees: a single central `.git/machete` for all worktrees (as up to 3.10) or a per-worktree `.git/worktrees/.../machete`
```